script/env/dolphinscheduler_env.sh

```diff
@@ -25,4 +25,4 @@ export HIVE_HOME=/opt/soft/hive
 export FLINK_HOME=/opt/soft/flink
 export DATAX_HOME=/opt/soft/datax/bin/datax.py
 
-export PATH=$HADOOP_HOME/bin:$SPARK_HOME1/bin:$SPARK_HOME2/bin:$PYTHON_HOME:$JAVA_HOME/bin:$HIVE_HOME/bin:$PATH:$FLINK_HOME/bin:$DATAX_HOME:$PATH
+export PATH=$HADOOP_HOME/bin:$SPARK_HOME1/bin:$SPARK_HOME2/bin:$PYTHON_HOME/bin:$JAVA_HOME/bin:$HIVE_HOME/bin:$PATH:$FLINK_HOME/bin:$DATAX_HOME:$PATH
```

Comment:
`PYTHON_HOME` needs to be set to `/usr`

Reply:
In this file, PYTHON_HOME and HADOOP_HOME have the same meaning, not the actual configuration but just telling the user to change to the installation directory of the program, so I set the value of PYTHON_HOME to /opt/soft/python instead of /usr